### ISSUE TITLE
formulation in PickConflict task

### ISSUE THREAD

**Issue by jvoigtlaender** [fmidue/modelling-tasks#121]
This:
> Which of the following Petri nets has exactly two transitions that are in conflict?

was taken by one student as suggesting there is a notion of "three transitions in conflict" and as if the task were (also) to check that a Petri net does not have three transitions in conflict.

Maybe a reformulation makes sense. For example "... has exactly one pair of transitions ...".

**Comment by jvoigtlaender** [comment]
This is already addressed by https://github.com/jvoigtlaender/modelling-tasks-internal/commit/cfb9e8a793117f3d030d266f0a26f7997658c741?

**Comment by jvoigtlaender** [comment]
Though that commit does it both for the `PickConflict` and the `PickConcurrency` task, whereas the issue here only talked about `PickConflict`. Deliberately?

**Comment by marcellussiegburg** [comment]
Yes. I thought the argument could hold for the formulation of concurrency in a similar way. Would it not?

**Comment by jvoigtlaender** [comment]
Yes, probably. Let's leave it at that for the moment. I guess I was "confused" because there also is https://github.com/jvoigtlaender/modelling-tasks-internal/issues/94, which makes it sound like there is even still consideration of whether the "exactly" should be written there (whereas actually I made it so a while ago: https://github.com/jvoigtlaender/modelling-tasks-internal/commit/e0f0cb3c5052bd9914c74a7fbe698421053c89b5).

One other consideration, though:

The new wording now appears in https://github.com/jvoigtlaender/modelling-tasks-internal/blob/cfb9e8a793117f3d030d266f0a26f7997658c741/src/Modelling/PetriNet/ConcurrencyAndConflict.hs#L191 and https://github.com/jvoigtlaender/modelling-tasks-internal/blob/cfb9e8a793117f3d030d266f0a26f7997658c741/src/Modelling/PetriNet/ConcurrencyAndConflict.hs#L214 but not in https://github.com/jvoigtlaender/modelling-tasks-internal/blob/cfb9e8a793117f3d030d266f0a26f7997658c741/src/Modelling/PetriNet/ConcurrencyAndConflict.hs#L199 and https://github.com/jvoigtlaender/modelling-tasks-internal/blob/cfb9e8a793117f3d030d266f0a26f7997658c741/src/Modelling/PetriNet/ConcurrencyAndConflict.hs#L222 which are also part of the task description. But maybe that is just fine, because then both old and new formulation are present in each task description, so each student sees both in case one is better understood than the other.
